scripts/in_container/run_tmux.sh

```diff
@@ -22,6 +22,12 @@ if [[ ${START_AIRFLOW:="false"} == "true" ]]; then
     export AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS=${LOAD_DEFAULT_CONNECTIONS}
     export AIRFLOW__CORE__LOAD_EXAMPLES=${LOAD_EXAMPLES}
 
+    # Use LocalExecutor if not set and if backend is not sqlite as it gives
```

Comment:
Added comment too @potiuk

Reply:
I also think the right executor should be set if you just enter breeze with --backend postgres or --backend mysql to shell rather than use 'start-airflow' (new feature)

Reply:
but won't that interfere when running tests using `pytest tests/some_file` inside `./breeze`

Reply:
i.e I feel there are 2 kind of uses currently I can think of

1) People who run `./breeze --backend postgres` to run tests so that they can replicate their CI failures for examples
2) People who want to run DAGs and try out new features/ test RCs or alphas etc for whom `./breeze start-airflow --backend postgres` would be easier because it supports using an ENV file too and I feel it is very powerful because of that and coz it also supports init file, WDYT ?

Reply:
So far it never interfered. The default executor for Postgres/Mysql was Local, for Sqlite it was Sequential (set by the variable in backend-sqlite.yml) . That's how it worked so far. somehow it looks like with the recent changes Sequential became the default ?

Reply:
Are you sure about that, I can remember that is was always SequentialExecutor for me inside of breeze but maybe not.

Reply:
> i.e I feel there are 2 kind of uses currently I can think of

Not really. People were running breeze and then started airflow inside. 'start-airflow' is very recent addition and only there to automate tmux terminals and starting scheduler/webserver.

Most people simply start breeze and manually running webseerver/scheduler and get it working there.

I believe making airflow not work by default in breeze is a very bad idea.

Reply:
>I believe making airflow not work by default in breeze is a very bad idea.

What do you mean? How will it make airflow not work

Reply:
the instructions to run RCs and test alphas are more recent and were added after `start-airflow` command is might point. Our CI and breeze update more frequently (multiple times a week), so if it makes sense we should promote and tell users to use new stuff  even if it is recent.

Reply:
so far when you run:

```
./breeze 
```

and then 

```
tmux
```

and then 


in one terminal - `airlfow webserver`

in another ``airlfow scheduler``

it just worked (because LocalExccutor was default for postgres/mysql).

seems that now default is the Sequential one - no matter what backend is used. Seems that if you do not set executor and the DB is postgres/mysql then the sequential one is used ?

Reply:
I am not sure if I am right about it - but it seems this is how it works now - we get sequential executor by default where we used to have local for Pg/Msql. Not sure where it came from, but it's just wrong default.

Reply:
Or maybe sequential executor worked well with PG/MG and stopped working well with it... either way it's something we need to fix (at the least by failing hard if sequential is wrong for PG/MYSQL but best - changing the default)

Reply:
that is correct, can you remember when it used to use `LocalExecutor` please before? I am honestly happy to do that and have approved your PR as long as you are sure it worked fine previously -- because I am not sure about that

Reply:
SequentialExecutor works with all the backends and have always worked with all backends, with SequentialExecutor tasks are just executed sequentially -- nothing more. Why do you think it would fail (except for the current bug)

That is the current behavior with Airflow too, the default executor is `SequentialExecutor` even if you used Postgres or MySQL until you change the executor separately too

Reply:
I really thing we should simply detect when the combination of DB/executor is not going to work and fail hard. I think this is the root cause of the problem.

Reply:
apart from the current bug why do you think SequentialExecutor does not work with MySQL or Postgres